### PR TITLE
Add positions section

### PR DESCRIPTION
**Pull request by hamzaibrahim21**
### TODO

- [x] Add `positions/index.md` file
- [x] Add `weight` to both publications and team to keep the menu order

### Status
- [x] Ready to go

### CODE REVIEW

**Comment by f-sod** [comment]
Thanks @hamzaibrahim21, It looks good to me.
@AndreaVolkamer, Michael does not figure in the table because he already has many students.

**Review by f-sod — APPROVED** [review]

File: content/positions/_index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Interested in working with us?
+date: 2021-03-03
+menu:
+    main:
+        pre: briefcase
+        name: Interested in working with us?
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:7]:
```suggestion
        name: Working with us?
```

File: content/positions/_index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+weight: 120
+nav: false
+---
+
+## Master and Bachelor Student / Student Assistant (HiWis)
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:12]:
```suggestion
## Master and Bachelor Students / Student Assistants (HiWis)
```

File: content/positions/_index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+At Volkamer Lab, we develop methods at the interface between structural bioinformatics and cheminformatics, mostly applied in the context of computer-aided drug design with a focus on machine learning methods. 
+In the table bellow you can have a view of the current work ongoing, for more details on each subject, please refer to the research section.
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:15]:
```suggestion
In the table bellow, you find an overview of the current topics in the group, for more details on each subject, please refer to the research section.
```

File: content/positions/_index.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+
+If you are interested in pursuing a bachelor’s or a master’s thesis with us, please gather the following information:
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:18]:
```suggestion
If you are interested in pursuing a bachelor’s or a master’s thesis with us, or are looking for an Hiwi job, please gather the following information:
```

File: content/positions/_index.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+- Transcript
+- CV
+- Previous thesis (if applicable)
+- A brief description (3-4 sentences) of what you want to work on in our group
+- Field of interest (refer to the table below) and specify the student supervisor in your email
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:23]:
```suggestion
- Field of interest (refer to the table below) - with concrete topic suggestions, if possible - and specify the most fitting supervisor in your email
```

File: content/positions/_index.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+Please write an email including the Volkamer Lab member you would like to work with and CC Paula (paula.kramer@cs.uni-saarland.de) and Floriane (floriane.odje@uni-saarland.de).
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:25]:
```suggestion
Please write an email including the Volkamer Lab member you would like to work with and CC Paula (paula.kramer[at]cs.uni-saarland.de) and Floriane (floriane.odje[at]uni-saarland.de), or our admin (stefanie.wessinger[at]uni-saarland.de). 
```

File: content/positions/_index.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+If you are interested in pursuing a bachelor’s or a master’s thesis with us, please gather the following information:
+- Transcript
+- CV
+- Previous thesis (if applicable)
+- A brief description (3-4 sentences) of what you want to work on in our group
+- Field of interest (refer to the table below) and specify the student supervisor in your email
+
+Please write an email including the Volkamer Lab member you would like to work with and CC Paula (paula.kramer@cs.uni-saarland.de) and Floriane (floriane.odje@uni-saarland.de). 
+
+**Important:** Do not use volkamer@cs.uni-saarland.de for Master’s/Bachelor’s HiWi position applications and inquiries.
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:27]:
```suggestion
**Important:** Do not use volkamer[at]cs.uni-saarland.de for Master’s/Bachelor’s/HiWi position applications and inquiries, we are collecting the information centrally. Also note that we are often overbooked with supervisions, thus, make sure that you approach us in advance, so we have time to consider your application in out planning.
```

File: content/positions/_index.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+|--------------------------------------|--------------------------------------------------------------------------------------------------|------------|
+| Paula Linh Kramer (PhD candidate)    | Deep learning, Fragment based drug design, generative AI                                         | SB         |
+| Floriane Odje (PhD candidate)        | Machine learning, Image analysis, toxicity and activity prediction, pattern identification                    | SB         |
+| Afnan Sultan (PhD candidate)         | Large language models (LLM), Generative AI for non-toxic properties, property prediction, explainable AI | SB         |
+| Yanyuan Zhang (PhD candidate)        | Structure based, binding site comparison, Software Development                                   | SB         |
+| Hamza Ibrahim (PhD candidate)        | Virtual screening, physics-based ML                                                              | SB         |
+| Dr. Raquel López-Ríos de Castro | Molecular Dynamics, Machine learning                                                             | B          |
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:36]:
```suggestion
| Dr. Raquel López-Ríos de Castro (PostDoc) | Molecular Dynamics, Machine learning                                                             | B          |
```

File: content/positions/_index.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+| Dr. Guillermo Pérez Hernández   | Molecular Dynamics, Software Development                                                         | B          |
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:37]:
```suggestion
| Dr. Guillermo Pérez Hernández (PostDoc)   | Molecular Dynamics, Software Development                                                         | B          |
```

File: content/positions/_index.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+| Antoine Lacour (Postdoc)             | Virtual Screening, Software Development                                                          | SB         |
```
Comment (AndreaVolkamer) [inline content/positions/_index.md:38]:
```suggestion
| Antoine Lacour (PostDoc)             | Virtual Screening, Software Development                                                          | SB         |
```

**Review by AndreaVolkamer — APPROVED** [review]
Thanks all!

**Pull request merged**